modules/logging/nopencoder.go

```diff
@@ -26,87 +26,87 @@ type nopEncoder struct{}
 
 // AddArray is part of the zapcore.ObjectEncoder interface.
 // Array elements do not get filtered.
-func (nopEncoder) AddArray(key string, marshaler zapcore.ArrayMarshaler) error { return nil }
+func (nopEncoder) AddArray(_ string, _ zapcore.ArrayMarshaler) error { return nil }
```

Comment:
I'm not sure I like this. It makes it harder to read the intent of the params, even if they aren't used _for now_.

Reply:
in fact, I also am not sure I like it.  Last time I did a large lint of a repo, I chose not to use this linter.  Would you like me to disable it?

Reply:
Disagreements with linters is why we've not used a lot of them yet, I think :sweat_smile:

If we have ways of disabling certain checks, maybe that's manageable.

Reply:
Configuring linters kinda one of my specialties and I am very happy to customize the configuration to suit Caddy's unique needs.

In this case I put those unused parameter lint fixes last so they're easy to revert because I linted github.com/cosmos/ibc-go recently and came to the same conclusion that you did :D